crowdin.yml

```diff
@@ -1,3 +1,4 @@
 files:
   - source: /src/i18n/locales/en-US.json
     translation: /src/i18n/locales/%two_letters_code%.json
+commit_message: "[ci skip]"
```

Comment:
should this be `[skip ci]` instead? i have always used this - but, if `[ci skip]` works, then great!

Reply:
```suggestion
commit_message: "[skip ci]"
```
You are right